src/extras/crosshair.js

```diff
@@ -28,11 +28,19 @@ Dygraph.Plugins.Crosshair = (function _extras_crosshair_closure() {
 
   var crosshair = function crosshair(opt_options) {
     this.canvas_ = document.createElement("canvas");
+    this.updateCanvasSize(0, 0)
```

Comment:
would `1, 1` work as well? bit afraid of hitting corner cases here

Reply:
I think size is not important until canvas is added to DOM. Moved to `activate` callback, where it can be set to graph actual size